metapool/amplipool.py

```diff
@@ -51,10 +53,17 @@ def assign_emp_index(plate_df, metadata):
     return plate_df
 
 
-def _load_emp_indices():
+def _load_emp_indices(seqtype):
     """Helper method to load EMP primer plates"""
-    fn = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data',
-                      'emp-16S-V4-515F-806R-parada-april.tsv')
+    if seqtype == '16S':
+        fn = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data',
+                          'emp-16S-V4-515F-806R-parada-april.tsv')
+    elif seqtype == '18S':
+        fn = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data',
+                          'Euk1391f_EukBr_Nov2016.tsv')
+    elif seqtype == 'ITS':
+        fn = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data',
+                          'ITS1f_ITS2_Nov2016.tsv')
```

Comment:
What do you think about consolidating all these in one single call to `os.path.join....`. And adding an error message if the seqtype is not a known value i.e. what would happen if none of the `if` clauses are True. For example like:

```suggestion
    if seqtype == '16S':
        tsv_name = 'emp-16S-V4-515F-806R-parada-april.tsv'
    elif seqtype == '18S':
        tsv_name = 'Euk1391f_EukBr_Nov2016.tsv'
    elif seqtype == 'ITS':
        tsv_name = 'ITS1f_ITS2_Nov2016.tsv'
    else:
        raise ValueError(f'Unrecognized value "{seqtype}" for seqtype')

    fn = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data',
                      tsv_name)
```